client/src/components/Navbar/Navbar.tsx

```diff
@@ -122,71 +139,134 @@ const Navbar: React.FC = () => {
-              </>
+        {!toggoleNavbar && (
+          <Grid container alignItems="center" gap={2} justifyContent="flex-end">
+            {renderMenuItems()}
+            {loggedInUser && (
+              <Grid xs={2} item>
+                <>
+                  <IconButton
+                    size="large"
+                    aria-label="account profile picture"
+                    aria-controls="menu-navbar"
+                    arais-haspopup="true"
+                    onClick={handleMenuOpen}
+                    color="inherit"
+                  >
+                    <img style={{ width: 50 }} src={`https://robohash.org/${loggedInUser.email}`} />
+                  </IconButton>
+                  <Menu
+                    id="menu-appbar"
+                    anchorEl={anchorEl}
+                    anchorOrigin={{
+                      vertical: 'bottom',
+                      horizontal: 'right',
+                    }}
+                    keepMounted
+                    transformOrigin={{
+                      vertical: 'top',
+                      horizontal: 'right',
+                    }}
+                    open={open}
+                    onClose={handleClose}
+                  >
+                    <DropdownMenuItem component={NavLink} to="/profile/settings" onClick={handleClose}>
+                      <ListItemIcon>
+                        <Settings fontSize="small" />
+                      </ListItemIcon>
+                      <ListItemText>Settings</ListItemText>
+                    </DropdownMenuItem>
+                    <DropdownMenuItem onClick={handleClose}>
+                      <ListItemIcon>
+                        <Person fontSize="small" />
+                      </ListItemIcon>
+                      <ListItemText>Profile</ListItemText>
+                    </DropdownMenuItem>
+                    <Divider />
+                    <DropdownMenuItem onClick={handleLogout}>
+                      <ListItemIcon>
+                        <Logout fontSize="small" />
+                      </ListItemIcon>
+                      <ListItemText>Logout</ListItemText>
+                    </DropdownMenuItem>
+                  </Menu>
+                </>
+              </Grid>
+            )}
+          </Grid>
+        )}
+        {toggoleNavbar && (
+          <>
+            <Grid container gap={2} sx={{ flexDirection: 'column', mt: 5 }}>
+              {renderMenuItems()}
+              {loggedInUser && (
+                <Grid xs={2} item>
+                  <>
+                    <IconButton
+                      size="large"
+                      aria-label="account profile picture"
+                      aria-controls="menu-navbar"
+                      arais-haspopup="true"
+                      onClick={handleMenuOpen}
+                      color="inherit"
+                    >
+                      <img style={{ width: 50 }} src={`https://robohash.org/${loggedInUser.email}`} />
```

Comment:
Please add the `alt` attribute. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader).